### ISSUE TITLE
adding export capabilities for correction projects

### ISSUE THREAD

**Issue by jvivaldi** [webanno/webanno#562]
I am involved in the creation of a POS tagged GoldStandard Corpus. The documents are tagged using Freeling (a multilingual an open source language analysis tool suite) and the output is revised using WebAnno. 
Currently a WebAnno correction project requires to correct wrong tokens and validate all the remaining POS and lemma information. Only after this operation it is possible to export the whole file.
I understand that this may be a design decition for quality control. This operation take some time and in my opinion after correcting a few documents its efficiency may no be high. 
My suggestion is to add an option (perhaps activated by the administrator) that should allow to keep this operation as it is or allow the user to do some correction and then automatically copy all the POS & lemma information  from the "Suggestion" to the "Annotation".
This should be a way to speed up the correction process.

**Comment by reckart** [comment]
@jvivaldi So the problem is not really the export capabilities but rather bulk-accepting information from the "suggestion" pane, right?

**Comment by seyyaw** [comment]
@reckart Yes, that is what @jvivaldi requires (from our email conversation). I remember we decide the user to click every annotations so that we know it is really corrected. Bulk accept, is still a bad Idea for me. Unlike the Automation project, documents in Correction under the "Suggestion" pane will not be changed (during the correction process). So, there is no point to copy the same information to the "Annotation" pane while they are the same (with the original document imported to the Correction project). Since the original "suggestion" annotation are not changed, it is possible to export modified "annotation" documents and manually (with external script) compare with the original "correction" document to bulk copy unchanged annotations.

**Comment by reckart** [comment]
Mind that the correction process is not necessarily limited to copying an annotation to the upper pane and changing it - it can also include adding or entirely removing annotations. In particular for the case of removing annotations, there is no you can figure out if an annotation should *not* be included in the result by comparing a partially filled-in annotation pane to the original annotations in the suggestion pane. That can only be done by copying *all* annotations from the suggestion to the annotation pane and then removing those that are undesired. 

But that's just from a principled point of view. Maybe what you suggest works for the specific case of @jvivaldi. I hope we get further feedback from @jvivaldi.

**Comment by jvivaldi** [comment]
@reckart: You are right in the sense that the suggestion is not to add a new export capability but a way speedup the correction process. The suggested procedure should work well at least for creating a corrected POS tagged corpus. In such cases, to remove completely an annotation is unprobable (assuming that the document has been properly tokenized). Normally the operator should change the POS tag (due to a POS tagging error) and sometimes the lemma (the word is not included in the POS tagger dictionary). 
My hypotheses is that forcing the user to click on each piece of information to transfer do not significantly improves the quality of the final result. After correcting a few documents the user can always mechanically accept all the informations without an effective check.
Of course the merging of the corrected information with the original text can be done with an external script but I think this is the type of task that the user expects will be solved by this tool.

**Comment by seyyaw** [comment]
@jvivaldi Some questions: 1) Assume you have an erroneous POS annotation (or lemma,...), which occur multiple times on the same token. Do you like correcting it throughout the document (when corrected the first time) when it corrected once? Should the annotator get confirmation for such annotation propagation or do it silently? 2) Do we always prefer to maintain annotations on the "Annotation" pane when auto-merging? What is your take here @ChrisBieTUDA ?

**Comment by jvivaldi** [comment]
@seyyaw regarding your questions:
1) is too risky to assume that multiple occurrences of the token should be always corrected in the same way. It may happen that in a given context it should be corrected but in another context it should not be corrected (imagine a word that may have different POS values according the context -ex. "run" as in "the boy run to the fence" vs. "the second run failed to finish"-). I suggest do not propagate it; the ideal way seems to be to signal the user in next occurrences of the same token (perhaps highlighting it).
2) I am not sure to understand what you mean. My original suggestion was to merge the annotation in the "Annotation" pane (it means with the user correction) with those in the "User suggestion" pane.
